### PR TITLE
Docs/readme

### PR DESCRIPTION
**Pull request by gustavosinbandera1**
added contributing PR steps

### CODE REVIEW

**Comment by gustavosinbandera1** [comment]
I have copy and paste PR steps from Kevin contributing file, I think is pretty clear to follow the steps

File: README.md [hunk 1]
```diff
@@ -2,20 +2,23 @@
   <a href="https://locha.io/">
   <img height="200px" src="doc/LogotipoTurpial-Color.20-09-19.svg">
   </a>
+  <br>
+ 
 </p>
 
 <p align="center">
   <a href="https://locha.io/">Project Website</a> |
+  <a href="https://btcven.github.io/turpial-firmware/">Code Documentation</a> |
```
Comment (jeandudey) [inline README.md]: This shouldn't be here, doesn't belong to radio-firmware.

File: README.md [hunk 2]
```diff
@@ -38,32 +41,7 @@ Here are some places if you want to support us:
...
-* [Getting started.](#getting-started)
-  - [Install SRecord.](#install-srecord)
-  - [Install build essentials (make, etc).](#install-build-essentials-make-etc)
-  - [Install ARM compiler.](#install-arm-compiler)
-  - [Install Uniflash.](#install-uniflash)
-  - [Compiling the project.](#compiling-the-project)
-
-[License.](#license)
-
-## Getting started.
```
Comment (jeandudey) [inline README.md]: Why the ToC was removed?

**Comment by jeandudey** [comment]
@gustavosinbandera1 resolve conflics and do the changes please.

**Pull request closed without merging**